reinforcement_learning/agents/basic_mc_agent/stochastic_model.py

```diff
@@ -75,47 +73,4 @@ def _repr_svg_(self):
         return self._get_graph()._repr_svg_()
 
     def view(self):
-        return self._get_graph().view()
-
-
-if __name__ == '__main__':
-    # Model test
-
-    m = StochasticModel()
-
-    s1 = SimpleState([[-1, 0], [-1, -1]])
-    a1 = SimpleAction([1, 0])
-    s2 = SimpleState([[-1, 0], [0, -1]])
-    s3 = SimpleState([[-1, 0], [-1, 0]])
-    m[s1, a1] = s2
-    m[s1, a1] = s3
-    m[s1, a1] = s3
-    m[s1, a1] = s3
-    m[s1, a1] = s3
-
-    a2 = SimpleAction([0, 0])
-    s4 = SimpleState([[0, 0], [0, -1]])
-    m[s2, a2] = s4
-
-    a4 = SimpleAction([1, 1])
-    s5 = SimpleState([[-1, 0], [0, 0]])
-    s6 = SimpleState([[-1, 0], [-1, 0]])
-    s7 = SimpleState([[-1, -1], [-1, -1]])
-    m[s2, a4] = s5
-    m[s2, a4] = s5
-    m[s2, a4] = s5
-    m[s2, a4] = s5
-    m[s2, a4] = s5
-    m[s2, a4] = s5
-    m[s2, a4] = s5
-    m[s2, a4] = s5
-    m[s2, a4] = s6
-    m[s2, a4] = s6
-    m[s2, a4] = s6
-    m[s2, a4] = s6
-    m[s2, a4] = s6
-    m[s2, a4] = s7
-    m[s2, a4] = s7
-    m[s2, a4] = s7
-
-    m.view()
```

Comment:
It's also necessary, because only human-eay can test if graphical appearience is good enaugh...